### PR TITLE
Migrate wavefront terraform plugin to v2.6.1 of the SDK

### PR DESCRIPTION
**Pull request by keep94**

### CODE REVIEW

File: wavefront/provider.go [hunk 1]
```diff
@@ -78,7 +78,48 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 
 }
 
-var wfMutexKV = mutexkv.NewMutexKV()
+// MutexKV is a simple key/value store for arbitrary mutexes. It can be used to
```
Comment (markpeek) [inline wavefront/provider.go:81]: This code should be put in its own file and commented as to its origin. In other words, best not to mix our code with HashiCorp code in the same file.

File: go.mod [hunk 2]
```diff
@@ -3,7 +3,7 @@ module github.com/vmware/terraform-provider-wavefront
 require (
 	github.com/WavefrontHQ/go-wavefront-management-api v1.14.0
 	github.com/google/go-cmp v0.5.2
-	github.com/hashicorp/terraform-plugin-sdk v1.11.0
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.4.3
```
Comment (markpeek) [inline go.mod:6]: Interesting your PR title says v2.6.1 but this file says v2.4.3. Not sure how that happened.

File: go.mod [hunk 3]
```diff
@@ -3,7 +3,7 @@ module github.com/vmware/terraform-provider-wavefront
...
 )
 
 go 1.13
```
Comment (markpeek) [inline go.mod:9]: In a separate PR, you might want to update to go 1.6.x and make a similar change to the [release workflow](https://github.com/vmware/terraform-provider-wavefront/blob/master/.github/workflows/release.yml#L20) to enable building providers for the M1 macs (arm64).

**Review by markpeek — CHANGES_REQUESTED** [review]
Added some comments for changes.

File: go.mod (hunk 2 above)
Comment (keep94) [reply go.mod:6]: Done.

File: wavefront/provider.go (hunk 1 above)
Comment (keep94) [reply wavefront/provider.go:81]: Done.

File: go.mod (hunk 2 above)
Comment (keep94) [reply go.mod:6]: Good catch. I upgraded to v2.6.1. The automated tools that I used must have been made when they were on v2.4.3

File: go.mod (hunk 3 above)
Comment (keep94) [reply go.mod:9]: Will do this in a separate PR.

**Review by markpeek — APPROVED** [review]
LGTM

**Pull request merged**